### PR TITLE
Adds preview

### PR DESCRIPTION
**Pull request by raftx24**
adds pdf preview

### CODE REVIEW

File: src/bulma/pages/files/components/File.vue [hunk 1]
```diff
@@ -62,6 +62,9 @@
         <url :show="temporaryLink !== ''"
             :link="temporaryLink"
             @close="temporaryLink = ''"/>
+        <Preview :file="preview"
```
Comment (aocneanu) [inline src/bulma/pages/files/components/File.vue]: lower `p`

File: src/bulma/pages/files/components/Preview.vue [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            <iframe :src="url"/>
+        </div>
+    </modal>
+</template>
+
+<script>
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faExternalLinkAlt } from '@fortawesome/free-solid-svg-icons';
+import { Modal } from '@enso-ui/bulma';
+import { Fade } from '@enso-ui/transitions';
```
Comment (aocneanu) [inline src/bulma/pages/files/components/Preview.vue]: we're not using Fade anywhere

File: src/bulma/pages/files/components/Preview.vue [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+            </div>
+            <iframe :src="url"/>
+        </div>
+    </modal>
+</template>
+
+<script>
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faExternalLinkAlt } from '@fortawesome/free-solid-svg-icons';
+import { Modal } from '@enso-ui/bulma';
```
Comment (aocneanu) [inline src/bulma/pages/files/components/Preview.vue]: in packages we want to import directly from deps, ot the aggregator (bulma).

we also need to make sure that the deps are added to package.json

**Review by aocneanu — COMMENTED** [review]
I tested it locally and it looks really good / is much faster to operate than new tab

File: src/bulma/pages/files/components/File.vue [hunk 4]
```diff
@@ -90,7 +90,7 @@ export default {
 
     directives: { tooltip: VTooltip },
 
-    components: { Confirmation, Url, Preview },
+    components: { Confirmation, Url, preview },
```
Comment (aocneanu) [inline src/bulma/pages/files/components/File.vue]: here it was ok:)

Comment (raftx24) [reply src/bulma/pages/files/components/File.vue]: ah yes, I need to fix import too.

**Pull request merged**